### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by saberlilyzz** [ChengBinJin/V-GAN-tensorflow#26]
What is the meaning of 1st_manual in training? Verification set? ?

**Comment by ChengBinJin** [comment]
@saberlilyzz 1st_manual and 2nd_manual means that the golden standard is annotated by different persons.
